Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -68,6 +72,19 @@ public float SafeZoneBuffer
             set => safeZoneBuffer = value;
         }
 
+        [SerializeField]
+        [Tooltip("Use raycasting when attaching to palm, instead of using palm joint directly.")]
```

Comment:
It's not clear what "use raycasting" means here -- does it mean casting a ray from the palm joint until the ray intersects something? Can you explaing in more detail, so that somebody can understand what this option does without needing to read the code?

Reply:
The hand safe zone target system uses raycasts to detect safe zones. This is the existing safe zone system; I don't fully agree with how it works but I wanted to keep the existing functionality as-is, if the user desires.

Reply:
This property has now been removed.